src/main_controller.ts

```diff
@@ -138,23 +139,15 @@ export class MainController implements vscode.Disposable {
         });
     }
 
-    private setClientInfo() {
-        readFile(path.posix.join(this.extensionPath, "package.json"))
-            .then((buffer) => {
-                const versionString = JSON.parse(buffer.toString()).version as string;
-                const [major, minor, patch] = [...versionString.split(".").map((n) => +n), 0, 0, 0];
-                this.client.setClientInfo("vscode-neovim", { major, minor, patch }, "embedder", {}, {});
-            })
-            .catch((err) => console.log(err));
+    public fireNvimEvent(event: string, ...args: VimValue[]): void {
```

Comment:
Where is this being used other than init?

Reply:
I struggled for a long time, not knowing where to put it.😂